src/api/cvc4cpp.h

```diff
@@ -3161,6 +3161,15 @@ class CVC4_PUBLIC Solver
    */
   std::vector<Term> getUnsatCore() const;
 
+  /**
```

Comment:
I don't think this shouldn't be exported yet.

We should maybe handle proofs analogously to how we handle models currently.

For now, this means that the command layer should retrieve the SmtEngine from the Solver when processing the GetProofCommand and store the result locally (as a string). See https://github.com/CVC4/CVC4/blob/master/src/smt/command.cpp#L1753.

Reply:
Sounds good. I've updated the PR